src/Xamarin.Android.Tools.Bytecode/ClassPath.cs

```diff
@@ -88,9 +88,6 @@ static bool ShouldLoadEntry (ZipArchiveEntry entry)
 			if (entry.Length == 0)
 				return false;
 
-			if (entry.Name == "module-info.class")
```

Comment:
Is this going to undo https://github.com/xamarin/java.interop/pull/1093?

That is, will this write `module-info.class` into `api.xml`?

Reply:
Yes, it will "undo" #1093, and No, it will *not* write `module-info.class` into `api.xml`.  (I should explicitly call this out in the commit message; doh!)

See `FixupModuleVisibility()`, in particular https://github.com/xamarin/java.interop/pull/1097/files#diff-cd606380b5886cb27bfcf4f255f042d4cb739a6a33545920a305cba1113dac74R386

We do two things in `FixupModuleVisibility()`:

 1. We collect all of the `ClassFile` entries of type `ClassAccessFlags.Module`, and *remove* them from `classFiles`.  This *prevents* them from being written into `api.xml`.
 2. For the `ClassFile` entries in (1), we find the set of exported packages (in `publicPackages`), and (2) iterate over every entry in `classFiles`.  Any `classFiles` entry with a package name *outside* of `publicPackages` has its visibility updated to `ClassAccessFlags.Internal`.

One problem with this approach, not considered until just now, is that if `class-parse` is parsing *multiple* `.jar` files and writing out a *single* `api.xml`, then the `module-info.class` information would be applied to entries *from a different `.jar`*.  This may result in "weird" behavior, and I'll need to think about this.